de-DE/Settings.js

```diff
@@ -0,0 +1,47 @@
+export default {
+  'Titles': {
+    'Session': 'Session-Einstellungen',
+    'System': 'System-Einstellungen',
+    'Window': 'Fenstereinstellungen',
+    'Accessibility': 'Einstellungen zur Barrierefreiheit',
+    'GameAndEmulator': 'Spiel- & Emulator-Einstellungen'
+  },
+  'Fields': {
+    'SessionName': {
+      'Titles': 'Session-Name',
```

Comment:
Is this really hyphenated in German?

Reply:
This is quite a deep topic in the German language, short answer: yes

Long answer: It depends on how accepted an anglicism is to the German language. Generally it is regarded to just rather hypenate the word connection, if you are unsure. In this particular case, session is a regularly used term in IT, but it isn't accepted enough into the German language to regard it as a native word. This distinction is a bit harder in the word Team-Management where even German dictionaries seem to argue about the correct way to spell it. Therefore I went for the safer approach to hyphenate it. If you're interested in the theory, this article explains it kind of nice: https://www.scribbr.de/wissenschaftliches-schreiben/zusammengesetzte-anglizismen/ (would need Google Translate I suppose)

Reply:
Dude as long as you're happy with it:P

Reply:
haha sorry, just my notorious curiousity 😄